### PR TITLE
WIP: docs: Add contributing files

### PR DESCRIPTION
**Pull request by deandreamatias**
@danvick Need a CONTACT METHOD to code of conduct. What can I put?

In other hand, maybe we can create a Slack, Discord, Telegram or something else to all community that use this packages

## Connection with issue(s)

Connected to #1038

## To Do

- [x] double check the original issue to confirm it is fully satisfied
- [ ] add testing notes and screenshots in PR description to help guide reviewers
- [ ] request the "UX" team perform a design review (if/when applicable)

### CODE REVIEW

**Comment by WilliamCunhaCardoso** [comment]
I believe a Discord would be interesting due to possibility to create different channels. A thing telegram cannot offer. About slack, I am not used to it, but am open to it.

File: packages/form_builder_phone_field/example/CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
+
+# Flutter form builder ecosystem Code of Conduct
```
[the remark this answers is not in the document]
Comment (deandreamatias) [reply packages/form_builder_phone_field/example/CODE_OF_CONDUCT.md]: Yeah! I agree

**Review by WilliamCunhaCardoso — APPROVED** [review]
Seems very good for me. Really great job!!

**Comment by danvick** [comment]
We used to have a Gitter channel, but it wasn't so popular, and that was when Github introduced the Discussions feature, and we decided to move over to it. Discord sure sounds like a good idea.

**Pull request merged**